vhive.go

```diff
@@ -71,19 +74,34 @@ func main() {
 	runtime.GOMAXPROCS(16)
 
 	rand.Seed(42)
-	snapshotter := flag.String("ss", "devmapper", "snapshotter name")
+
 	debug := flag.Bool("dbg", false, "Enable debug logging")
+	isMetricsMode = flag.Bool("metrics", false, "Calculate UPF metrics")
+	criSock = flag.String("criSock", "/etc/vhive-cri/vhive-cri.sock", "Socket address for CRI service")
+	sandbox := flag.String("sandbox", "firecracker", "Sandbox tech to use, valid options: firecracker, gvisor")
 
+	// Funcpool
 	isSaveMemory = flag.Bool("ms", false, "Enable memory saving")
-	isSnapshotsEnabled = flag.Bool("snapshots", false, "Use VM snapshots when adding function instances")
-	isUPFEnabled = flag.Bool("upf", false, "Enable user-level page faults guest memory management")
-	isMetricsMode = flag.Bool("metrics", false, "Calculate UPF metrics")
 	servedThreshold = flag.Uint64("st", 1000*1000, "Functions serves X RPCs before it shuts down (if saveMemory=true)")
 	pinnedFuncNum = flag.Int("hn", 0, "Number of functions pinned in memory (IDs from 0 to X)")
+
+	// Snapshotting
+	isSnapshotsEnabled = flag.Bool("snapshots", false, "Use VM snapshots when adding function instances")
+	isSparseSnaps = flag.Bool("sparsesnaps", false, "Makes memory files sparse after storing to reduce disk utilization")
+	isFullLocal = flag.Bool("fulllocal", false, "Use improved full local snapshotting")
+	snapsCapacityMiB = flag.Int64("snapcapacity", 102400, "Capacity set aside for storing snapshots (Mib)")
```

Comment:
```suggestion
	snapsStorageSize = flag.Int64("snapcapacity", 102400, "Total storage reserved for snapshots (GB)")
```

Please rename and define in GB (I assume you meant total storage, not per snap).